cluster/addons/storage-class/gce/default.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: extensions/v1beta1
+kind: StorageClass
+metadata:
+  name: default
+  annotations:
+    storage.beta.kubernetes.io/default-class: "true"
+  labels:
+    kubernetes.io/cluster-service: "true"
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-ssd
```

Comment:
As long as AWS stays with SSD I don't particularly mind (particularly since the whole point is that it is now easily changeable).  I think most cloud users that use volumes are running databases, where they typically value IOPS more than Gigabytes.  For people on that axis SSD is a better choice :-)